examples/dice/instrumented/init.sh

```diff
@@ -3,4 +3,4 @@
 
 #!/bin/bash
 
-module go.opentelemetry.io/contrib/examples/dice/instrumented
\ No newline at end of file
+go mod init go.opentelemetry.io/contrib/examples/dice/instrumented
```

Comment:
@svrnm, you probably want this to match what is in https://opentelemetry.io/docs/languages/go/getting-started/#setup, right?

However, maybe you prefer to have this PR merged to avoid gold-plating and start testing integration with https://github.com/open-telemetry/opentelemetry.io in practice?

Reply:
yes, it does not have to be feature perfect, there will be tweaks and minor changes here and there anyways.